test/utils.jl

```diff
@@ -0,0 +1,136 @@
+using ADNLPModels, NLPModels, NLPModelsJuMP, OptimizationProblems
+
+function set_meta(all::AbstractVector{T}) where T <: Union{Symbol, String}
+  for name in string.(all)
+    io = open("src/Meta/" * name * ".jl", "w")
+    write(io, generate_meta(name))
+    close(io)
+  end
```

Comment:
We should have a way of reusing the parts of the meta that have to be manually informed, though this can be done later.

Reply:
Sure, I can add another function like this one that replaces the name of some entries without replacing the whole meta.

Reply:
`replace_entry_meta([:AMPGO02], :everywhere_defined, :defined_everywhere)`